tpu/bootstrap.sh

```diff
@@ -0,0 +1,87 @@
+#! bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configures a single-client v2-8 TPU VM for DTensor.
+
+# This command produces cluster-run.sh, which can be used to run command
+# on the TPU VM.
+#
+# The git repo is cloned to the VMs.
+
+NAME="${USER}-dtensor-tpu-test"
```

Comment:
maybe also add the topology to the name.

Reply:
Done.